src/tests/AsyncTests.fs

```diff
@@ -200,18 +200,22 @@ open Fable.Core
 
 [<Test>]
 let ``Interaction between Async and Promise works``() =
-    let res = ref false
-    async { res := true }
-    #if FABLE_COMPILER
-    |> Async.StartAsPromise
-    |> Async.AwaitPromise
-    #else
-    |> Async.StartAsTask
-    |> Async.AwaitTask
-    #endif
-    |> Async.StartImmediate
-    equal true !res
-    
+    async {
+        let res = ref false
+        async { res := true }
+        #if FABLE_COMPILER
+        |> Async.StartAsPromise
+        |> Async.AwaitPromise
+        #else
+        |> Async.StartAsTask
+        |> Async.AwaitTask
+        #endif
+        |> Async.StartImmediate
+        // some wait is needed on Linux for some reason
```

Comment:
This is a bit odd - the basic idea of `StartImmediate` is that it starts the work and runs it synchronously until first "async" operation happens inside it - there is no "async" operation in `res := true`, so it would be good to understand what's going on here....